server/opts.go

```diff
@@ -3253,6 +3258,16 @@ func parseTLS(v interface{}) (t *TLSConfigOpts, retErr error) {
 			}
 			tc.Verify = verify
 			tc.Map = verify
+		case "verify_and_implicit_allow":
+			verify, ok := mv.(bool)
+			if !ok {
+				return nil, &configErr{tk, "error parsing tls config, expected 'verify_and_map' to be a boolean"}
```

Comment:
`verify_an_map` is not correct field name.

Reply:
fixed